ql/src/test/results/clientnegative/create_table_stored_by_invalid3.q.out

```diff
@@ -0,0 +1 @@
+FAILED: SemanticException Cannot find class 'ICEBERG'
```

Comment:
Thanks for checking this out, the message looks clean and precise now.
Can you also handle case this ('ICEBERG') to keep the message consistent? We can have something like `Unrecognized storage handler in STORED BY clause: 'ICEBERG'. Supported types = ICEBERG or FQCN of a storage handler.`

This should be an indicator enough for the user to remove the quotes. WDYT

Reply:
Thanks for sharing your thought. Now, I also think your idea sounds more informative. Let me revise it

Reply:
Done
https://github.com/apache/hive/pull/4954/commits/136035e9704ea9cd676d0722b83dbd4ee0310039